cryptol-saw-core/saw/Cryptol.sawcore

```diff
@@ -945,6 +945,9 @@ PRoundRational =
 PLiteral : (a : sort 0) -> sort 0;
 PLiteral a = Nat -> a;
 
+PLiteralLessThan : (a : sort 0) -> sort 0;
+PLiteralLessThan a = Nat -> a;
```

Comment:
I'm not entirely sure about the use of `Nat` here, since the `n` in `LiteralLessThan n a` can be `inf`. Should this be `Num` instead?

Reply:
No, we'll only ever ask for literals of finite values, even if the upper bound is `inf` (NB, it's an exclusive upper bound!)